doc/classes/Timer.xml

```diff
@@ -42,7 +42,7 @@
 	<members>
 		<member name="autostart" type="bool" setter="set_autostart" getter="has_autostart" default="false">
 			If [code]true[/code], the timer will start immediately when it enters the scene tree.
-			[b]Note:[/b] After the timer enters the tree, this property is automatically set to [code]false[/code].
+			[b]Note:[/b] After the timer enters the tree, this property is automatically set to [code]false[/code]. If the timer will be the part of the scene currently opened in the editor (for example, adding and running timer from [code]@tool[/code] script), this property will not set to [code]false[/code] and the timer will not start immediately when it enters the scene tree.
```

Comment:
Can't it just be...?
```suggestion
			[b]Note:[/b] After the timer enters the tree, this property is automatically set to [code]false[/code].
			[b]Note:[/b] This property does nothing when the timer is running in the editor.
```

Reply:
Sure!
This variant will look like my first version of changes, which I rejected as "not fully descriptive" :D